src/github.com/couchbase/sync_gateway/db/design_doc.go

```diff
@@ -117,27 +101,20 @@ func (db *Database) QueryDesignDoc(ddocName string, viewName string, options map
 
 // Cleans up the Value property, and removes rows that aren't visible to the current user
 func filterViewResult(input walrus.ViewResult, user auth.User) (result walrus.ViewResult) {
-	checkChannels := false
-	var visibleChannels ch.TimedSet
-	if user != nil {
-		visibleChannels = user.InheritedChannels()
-		checkChannels = !visibleChannels.Contains("*")
-	}
+
```

Comment:
It looks like we're still calling filterViewResult if the admin is querying a non-internal doc.  Is this needed?  We don't intend to do any filtering, and I'm not sure there's a reason to create a new walrus.ViewRow like we're doing below (instead of using the existing one as-is).  It also seems like it could break if the row.Value wasn't a string.  Looking back at the previous version of the design_doc.go, I had thought we could just leave out the 'isInternalDDoc' condition altogether?